ui/src/copyable_text.tsx

```diff
@@ -88,30 +92,34 @@ export const XCopyableText = ({ model }: { model: CopyableText }) => {
   React.useEffect(() => () => window.clearTimeout(timeoutRef.current), [])
 
   return (
-    <div
+    <Fluent.TextField
       data-test={name}
-      className={multiline ? css.multiContainer : css.compactContainer}
-      style={heightStyle as React.CSSProperties}
-    >
-      <Fluent.TextField
-        componentRef={ref}
-        value={value}
-        label={label}
-        multiline={multiline}
-        styles={{
-          root: { ...heightStyle, width: pc(100) },
-          wrapper: heightStyle,
-          fieldGroup: heightStyle || { minHeight: height },
-          field: { ...heightStyle, height, resize: multiline ? 'vertical' : 'none', },
-        }}
-        readOnly
-      />
-      <Fluent.PrimaryButton
-        title='Copy to clipboard'
-        onClick={onClick}
-        iconProps={{ iconName: copied ? 'CheckMark' : 'Copy' }}
-        className={clas(css.btn, copied ? css.copiedBtn : '', multiline ? css.btnMultiline : '')}
-      />
-    </div>
+      componentRef={ref}
+      value={value}
+      label={label}
```

Comment:
Probably not necessary in favor of `onRenderLabel`.
```suggestion
```

Reply:
Good point!